src/model/data.types.model.ts

```diff
@@ -74,3 +74,5 @@ export const IsSecureRegisterKey = "is_secure_register";
 export const PublicRegisterNameKey = "public_register_name";
 export const RegistrationNumberKey = "registration_number";
 
+// Update Journey
+export const OeNumber = "oe_number";
```

Comment:
Should we rename this to OeNumberKey to be consistent with the naming convention followed in this file

Reply:
yes you are right! Will do that now

Reply:
done!